server/etcdserver/api/v2store/store_ttl_test.go

```diff
@@ -132,7 +132,7 @@ func TestStoreUpdateDirTTL(t *testing.T) {
 	assert.False(t, e.Node.Dir)
 	assert.Equal(t, eidx, e.EtcdIndex)
 	e, _ = s.Get("/foo/bar", false, false)
-	assert.Equal(t, "", *e.Node.Value)
+	assert.Empty(t, *e.Node.Value)
```

Comment:
Could you please open a PR with this and `tests/e2e/v2store_deprecation_test.go`'s changes? I believe we can merge them into main, and the linters still pass (I tested it out). Thanks!